dagger/config_finder/config_processor.py

```diff
@@ -45,7 +45,10 @@ def process_pipeline_configs(self):
 
             _logger.info("Processing config: %s", config_path)
             config_dict = self._load_yaml(config_path)
-            pipeline = Pipeline(pipeline_config.directory, config_dict)
+            if config_dict:
+                pipeline = Pipeline(pipeline_config.directory, config_dict)
+            else:
+                continue
```

Comment:
It would be a good idea to add a log message here, just like when a job is disabled.